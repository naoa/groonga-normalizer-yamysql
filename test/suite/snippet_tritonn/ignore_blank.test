register normalizers/yamysql
register function/snippet_tritonn

table_create Entries TABLE_NO_KEY
column_create Entries content COLUMN_SCALAR ShortText

table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigramIgnoreBlank --normalizer NormalizerYaMySQLPartofspeechHtml
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content

load --table Entries
[
{"content": "テストの終わり"}
]

select Entries \
  --output_columns 'snippet_tritonn(content, 200, 2, \
  "NormalizerYaMySQLPartofspeechHtml", \
  1, 1, "...", "...", \
  "テスト終わり", "<span class=\\"w1\\">", "</span>"), content' \
  --command_version 2 \
  --match_columns 'content' \
  --query "テスト終わり"

select Entries \
  --output_columns 'snippet_tritonn(content, 200, 2, \
  "NormalizerYaMySQLPartofspeechHtml", \
  1, 1, "...", "...", \
  "テストが終わり", "<span class=\\"w1\\">", "</span>"), content' \
  --command_version 2 \
  --match_columns 'content' \
  --query "テストが終わり"
