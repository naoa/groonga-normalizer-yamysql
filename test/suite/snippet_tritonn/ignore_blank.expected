register normalizers/yamysql
[[0,0.0,0.0],true]
register function/snippet_tritonn
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigramIgnoreBlank --normalizer NormalizerYaMySQLPartofspeechHtmlProlong
[[0,0.0,0.0],true]
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content
[[0,0.0,0.0],true]
load --table Entries
[
{"content": "テストの終わり"}
]
[[0,0.0,0.0],1]
select Entries   --output_columns 'snippet_tritonn(content, 200, 2,   "NormalizerYaMySQLPartofspeechHtmlProlong",   1, 1, "...", "...",   "テスト終わり", "<span class=\\"w1\\">", "</span>"), content'   --command_version 2   --match_columns 'content'   --query "テスト終わり"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "snippet_tritonn",
          "Bool"
        ],
        [
          "content",
          "ShortText"
        ]
      ],
      [
        [
          "...<span class=\"w1\">テストの終わり</span>..."
        ],
        "テストの終わり"
      ]
    ]
  ]
]
